shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -312,11 +312,12 @@ void FlatlandExternalViewEmbedder::SubmitFrame(
         child_transforms_.emplace_back(
             flatland_layers_[flatland_layer_index].transform_id);
 
-        // Attach full-screen hit testing shield.
+        // Attach full-screen hit testing shield. Note that since the hit-region
+        // may be transformed (translated, rotated), we do not want to set
+        // width/height to FLT_MAX. This will cause a numeric overflow.
         flatland_->flatland()->SetHitRegions(
             flatland_layers_[flatland_layer_index].transform_id,
-            {{{0, 0, std::numeric_limits<float>::max(),
-               std::numeric_limits<float>::max()},
+            {{{0, 0, 1'000'000, 1'000'000},
```

Comment:
Can we keep this in a constexpr 'kMaxHeight/Width' ?

Reply:
Fixed.